harry-core/src/harry/model/SelectHelper.java

```diff
@@ -271,6 +275,18 @@ public static ResultSetRow resultSetToRow(SchemaSpec schema, OpSelectors.Monoton
         System.arraycopy(result, partitionKey.length + clusteringKey.length, staticColumns, 0, staticColumns.length);
         System.arraycopy(result, partitionKey.length + clusteringKey.length + staticColumns.length, regularColumns, 0, regularColumns.length);
 
+
+        List<Long> visited_lts_list;
+        if (schema.trackLts)
+        {
+            visited_lts_list = (List<Long>) result[result.length - 1];
+            visited_lts_list.sort(Long::compare);
```

Comment:
Good point. But since we _mostly_ use these to compare with actually visited LTS (which are in monotonically increasing order), we are sorting them here. I could not find a use-case for knowing the order things were inserted in. Maybe we can have a more sophisticated checker that can check query bounds / visibility or something.